myuw/views/api/instructor_schedule.py

```diff
@@ -108,6 +108,7 @@ def set_section_grading_status(section, person):
         else:
             raise
     except Exception:
+        return "error"
```

Comment:
Do you really mean to return before log_exception?

Reply:
Good catch, fixed